Sources/AWSLambdaEvents/DynamoDB.swift

```diff
@@ -15,7 +15,7 @@
 import struct Foundation.Date
 
 // https://docs.aws.amazon.com/lambda/latest/dg/with-ddb.html
-public struct DynamoDB {
+public enum DynamoDB {
```

Comment:
that is honestly crazy... a formatter changing types just because it feels like it?

Reply:
yea this is nasty imo - is there an option to turn this off?

Reply:
Actually I think this pretty clever... It detect's that this is used as a namespace and makes it explicit:

`enumNamespaces` -> Converts types used for hosting only static members into enums.
https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#enumnamespaces

Reply:
It's NOT up to a formatting tool to decide how I'm designing and using types.

We really should not allow this.

Reply:
I agree, its clever, but it should a human decision to change a type not a formatting tool

Reply:
I disabled the matching `swiftformat` rule. Since this change is actually clever in this case, do we want to keep it?

Reply:
changing an `enum` to a `struct` is also API breaking...

Reply:
@weissi we go the other way though. struct -> enum. just out of curiosity: in which way would enum -> struct be source breaking, if it is just about a namespace?

Reply:
```
import enum Foo.Bar
```

will break. No tool should ever switch `enum` to or from `struct` (or class or something). The programmer needs to decide.